test/blackbox-tests/test-cases/pkg/depopts/gh11058.t

```diff
@@ -0,0 +1,13 @@
+Reproduce github issue #11058
+
+Handling of more than one depopt:
+
+  $ . ../helpers.sh
+
+  $ mkpkg bar <<'EOF'
+  > depopts: [ "a" "b" "c" ]
```

Comment:
Could you add a test case with a nested `or`? I've been trying to break this with constructs like

```
depopts: [ (("e" | "a") | ("d" | "f")) "b" "c" ]
```

and it works, but I think a case that shows arbitrary nesting works is nice to have have to avoid regressions.

Reply:
Good idea. Done